shared/subspace-logging/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "subspace-logging"
+description = "Ensure all logging uilities are centralized for the whole project"
+license = "Apache-2.0"
+version = "0.0.1"
+authors = []
```

Comment:
You can put your name here, or copy the authors from one of the places you moved the code from.